scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -144,7 +144,7 @@
     <div href="#" class="bg-dark list-group-item text-white">
       <div class="d-flex flex-row bd-highlight mt-1">
         <div>
-          <span class="menu-collapsed">{{report_key|replace("_"," ")|title}}</span>
+          <span class="menu-collapsed">{{report_key|replace("_"," ")|title|upper_na}}</span>
```

Comment:
I just couldn't have either "Rna report" or make the key names capital RNA... 😅

Reply:
It works fine and it shows:

<img width="199" alt="image" src="https://github.com/user-attachments/assets/b535248d-e502-40c4-9d82-464a4a55154f">

But wound't it be better if we placed it close to the delivery report, or to the gene fusion one?

I would have this order, perhaps

- General report
- mtDNA report
- Delivery report
- RNA Delivery report
- Gene fusion report
- Pipeline version
- Reference info

I know that many cases will lack some of these report, but feels more ordered this way.